src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -369,6 +369,33 @@ private static NamedTypeSymbol GetTupleUnderlyingType(ImmutableArray<TypeSymbol>
         }
 
         /// <summary>
+        /// For tuples with no natural type, we still need to verify that an underlying type of proper arity exists, and report if otherwise.
+        /// </summary>
+        internal static void VerifyTupleTypePresent(int cardinality, CSharpSyntaxNode syntax, CSharpCompilation compilation, DiagnosticBag diagnostics)
+        {
+            int remainder;
+            int chainLength = NumberOfValueTuples(cardinality, out remainder);
+
+            NamedTypeSymbol firstTupleType = compilation.GetWellKnownType(GetTupleType(remainder));
+
+            if ((object)diagnostics != null && (object)syntax != null)
```

Comment:
Same for syntax.

---
In reply to: [63761762](https://github.com/dotnet/roslyn/pull/11389#discussion_r63761762) [](ancestors = 63761762)